db/migrate/20180321152146_add_locale_to_dimensions_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddLocaleToDimensionsItems < ActiveRecord::Migration[5.1]
+  def change
+    add_column :dimensions_items, :locale, :string
```

Comment:
Locale should be `not null`. In this PR, I would default it to `en`, although we would need to move the initialiser into the model.